### DISCUSSION TITLE
发现信息图曾经有个 feature

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#616] [General]
点号会被识别成项目符号，不知道是怎么出现的，也不知道是怎么消失的，大概和 v-html 有关，心海的 json 里面就是点号
![image](https://user-images.githubusercontent.com/9006264/151010604-0524605c-af23-4390-be45-673cc2d65744.jpeg)
![image](https://user-images.githubusercontent.com/9006264/151010612-1928007e-bde4-4dac-922e-a65fb8a54494.jpeg)

**Comment by mark9804** [comment]
Done

----
这么一看好像忘记对齐文字边缘了

`texi-align: justify`

**Comment by Xm798** [comment]
说起这个东西……我爬信息的时候发现用元素战技替换元素战技名字替换不掉，然后发现技能的点是`•`，命座的点是`·`……

**Reply by mark9804** [reply]
据我所知点有三种 `•・·`

**Comment by Xm798** [comment]
p2是不是原来的那套json？

**Comment by mark9804** [comment]
草，破案了，json 真有改过，我差点当作 vue 的 bug 提上去了
![image](https://user-images.githubusercontent.com/9006264/151105475-ad7f07cd-2b96-4a35-84e5-0e3773e825bc.png)

**Reply by Xm798** [reply]
之前那套json是原作者手动阅读理解精简过的……现在没有篇幅限制了就直接采用了官方描述，避免造成可能的误解吧（懒……）
